build_tools/benchmarks/suites/cmake_rule_generator.py

```diff
@@ -25,6 +25,8 @@
     open(TEMPLATE_DIR / "iree_tflite_import_template.cmake", "r").read())
 TF_IMPORT_CMAKE_TEMPLATE = string.Template(
     open(TEMPLATE_DIR / "iree_tf_import_template.cmake", "r").read())
+IREE_BYTECODE_MODULE_CMAKE_TEMPLATE = string.Template(
+    open(TEMPLATE_DIR / "iree_bytecode_module_template.cmake", "r").read())
```

Comment:
Doesn't this leave the file handle open? You could extract a function here that does things with a conext manager

```python
def template_from_file(basename):
  with open(TEMPLATE_DIR / basename, "r") as f:
    return string.Template(f.read())

IREE_BYTECODE_MODULE_CMAKE_TEMPLATE = template_from_file("iree_bytecode_module_template.cmake")
```

Reply:
Done.

I thought the lifetime of file handle would end when the `read()` completes, so it wouldn't leak into the global scope. But using a context manager is much clearer.

Reply:
Yeah I'm not totally sure that it doesn't. I think it will get closed when Python garbage collects, and it may even do so aggressively on EOF, but I think using the context manager is cleaner indeed.